src/App.js

```diff
@@ -1,25 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import "./App.css"
+import Header from './Header'
+import Todolist from './Todolist'
 
-function App() {
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <>
+    <div className='daddyDiv'>
+    <div className='mainTodoDiv'>
```

Comment:
css classes should be unserscored or dashed 
mainTodoDiv => main_todo_div/main-todo-div @Kunal-inno